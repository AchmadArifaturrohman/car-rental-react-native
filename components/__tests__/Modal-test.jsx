import * as React from "react";
import renderer from "react-test-renderer";

import ModalPopup from "../Modal";

it(`renders correctly`, () => {
  const tree = renderer
    .create(<ModalPopup visible={true}>Snapshot test!</ModalPopup>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
