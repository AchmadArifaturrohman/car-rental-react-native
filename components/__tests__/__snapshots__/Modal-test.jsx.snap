// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Modal
  animationType="fade"
  hardwareAccelerated={false}
  transparent={true}
  visible={true}
>
  <View
    style={
      [
        {
          "backgroundColor": "#fff",
          "borderColor": "#000000",
          "shadowColor": "rgba(0, 0, 0, 0.5)",
        },
        {
          "alignItems": "center",
          "backgroundColor": "rgba(0, 0, 0, 0.7)",
          "flex": 1,
          "justifyContent": "center",
        },
      ]
    }
  >
    Snapshot test!
  </View>
</Modal>
`;
